otter/cloud_client/rcv3.py

```diff
@@ -167,7 +168,8 @@ def _check_bulk_add(attempted_pairs, result):
         match = _NODE_ALREADY_A_MEMBER_PATTERN.match(error)
         if match is not None:
             pair = match.groupdict()
-            exists = exists.add((pair["lb_id"], pair["node_id"]))
+            # lb_id returned in upper case
```

Comment:
are lower-case/upper-case equivalent?  when is it returned in upper case?  this comment could use some elaboration.